lib/galaxy_test/api/test_datasets.py

```diff
@@ -200,3 +201,61 @@ def test_update_datatype(self):
             f"histories/{self.history_id}/contents/{hda_id}",
             data={'datatype': 'invalid'}, json=True)
         self._assert_status_code_is(invalidly_updated_hda_response, 400)
+
+    def test_delete_batch(self):
+        num_datasets = 4
+        dataset_map: Dict[int, str] = {}
+        history_id = self.dataset_populator.new_history()
+        for index in range(num_datasets):
+            hda = self.dataset_populator.new_dataset(history_id)
+            dataset_map[index] = hda["id"]
+
+        expected_deleted_ids = [dataset_map[1], dataset_map[2]]
+        delete_payload = {
+            "ids": expected_deleted_ids
+        }
+        delete_response = self._delete("datasets", data=delete_payload, json=True)
+        self._assert_status_code_is_ok(delete_response)
+        deleted_result = delete_response.json()
+
+        assert deleted_result["success_count"] == len(expected_deleted_ids)
+        for purged_id in expected_deleted_ids:
+            dataset = self._get(f"histories/{history_id}/contents/{purged_id}").json()
```

Comment:
```suggestion
        for deleted_id in expected_deleted_ids:
            dataset = self._get(f"histories/{history_id}/contents/{deleted_id}").json()
```
Assuming that was the intention?

Reply:
It was! thanks! :)